<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="fontstyle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>样式</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="CheckBoxUnderline">
      <property name="text">
       <string>Underline</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CheckBoxItalic">
      <property name="text">
       <string>Italic</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CheckBoxBold">
      <property name="text">
       <string>Bold</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="color">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>394</width>
     <height>48</height>
    </rect>
   </property>
   <property name="title">
    <string>颜色</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButtonBlue">
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_Red">
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_Yellow">
      <property name="text">
       <string>Yellow</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_Green">
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string>Hello world!
today is 27th June ,Monday!</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>355</width>
     <height>55</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="BntOK">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BntCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="BntClose">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>CheckBoxUnderline</tabstop>
  <tabstop>CheckBoxItalic</tabstop>
  <tabstop>CheckBoxBold</tabstop>
  <tabstop>radioButtonBlue</tabstop>
  <tabstop>radioButton_Red</tabstop>
  <tabstop>radioButton_Yellow</tabstop>
  <tabstop>radioButton_Green</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>BntOK</tabstop>
  <tabstop>BntCancel</tabstop>
  <tabstop>BntClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>BntOK</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BntCancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BntClose</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
